### PR TITLE
Collect stdout & stderr from Command

### PR DESCRIPTION
**Pull request by tasleson**
Addresses https://github.com/stratis-storage/devicemapper-rs/issues/256

Signed-off-by: Tony Asleson <tasleson@redhat.com>

### CODE REVIEW

**Comment by agrover** [comment]
LGTM :+1:

File: src/test_lib.rs [hunk 1]
```diff
@@ -38,6 +40,41 @@ pub fn test_string(name: &str) -> String {
     namestr
 }
 
+/// Execute command while collecting stdout & stderr.
+pub fn execute_cmd(cmd: &mut Command, error_msg: &str) -> DmResult<()> {
+    let result = cmd.output().unwrap();
```
Comment (mulkieran) [inline src/test_lib.rs]: Might as well just use "?" here. All these methods seem to be designed to return errors, except for ```get_dm```, which is anomalous and should be fixed so it behaves like the version in stratisd, which doesn't unwrap.

Comment (tasleson) [reply src/test_lib.rs]: Indeed, originally I didn't have the functions returning anything, but I switched them to return a result and I forgot to remove the unwrap, thanks!

File: src/test_lib.rs [hunk 2]
```diff
@@ -38,6 +40,55 @@ pub fn test_string(name: &str) -> String {
     namestr
 }
 
+/// Execute command while collecting stdout & stderr.
+pub fn execute_cmd(cmd: &mut Command, error_msg: &str) -> DmResult<()> {
```
Comment (mulkieran) [inline src/test_lib.rs]: Try leaving out the ```error_msg``` parameter and just printing out the command that failed.

Sort of like in https://github.com/stratis-storage/stratisd/pull/988/commits/4050571d312b15e2ed04db5dacd0b5ad5b35d176.

This is only test code, but I think the principle still applies.

**Pull request merged**